explorer/client/src/components/Table.tsx

```diff
@@ -71,34 +77,48 @@ interface IProps extends WithStyles<typeof styles> {
   emptyMsg?: string
 }
 
+const renderCells = ({ r, idx }: { r: any[]; idx: number; }) => (
+  <TableRow key={idx}>
+    {r.map((col: Column, idx: number) => (
+      <TableCell key={idx} column={col} />
+    ))}
+  </TableRow>
+)
+
 const renderRows = ({ headers, rows, loadingMsg, emptyMsg }: IProps) => {
   if (!rows) {
     return <Loading colCount={headers.length} msg={loadingMsg} />
   } else if (rows.length === 0) {
     return <Empty colCount={headers.length} msg={emptyMsg} />
   } else {
     return rows.map((r: any[], idx: number) => (
-      <TableRow key={idx}>
-        {r.map((col: Column, idx: number) => (
-          <TableCell key={idx} column={col} />
-        ))}
-      </TableRow>
+      <>
+        <Hidden xsDown>{renderCells({ r, idx })}</Hidden>
+        <Hidden smUp>{renderCells({ r: [r[2], r[1], r[3], r[0]], idx: idx })}</Hidden>
```

Comment:
@rupurt Would you like to see Constants for indices?

Reply:
@NavyAdmiral I feel like this logic doesn't belong in `<Table>` as it's meant to be a generic component. We should be using `<Hidden>` to render different `<Table>` components in `JobRuns/List`.